.storybook/components/IconGrid/IconGrid.css

```diff
@@ -3,9 +3,9 @@
 \*------------------------------------*/
 
 /**
- * 1) The icon grid is not for production and is meant solely to display
```

Comment:
I was wondering why some of these ordered lists don't even reference anything in the code 🤔

Reply:
Yeah it's kinda confusing because I sometimes read through the whole CSS block to see if the number is in there somewhere 🤷